hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -2199,6 +2199,23 @@
                 <xs:complexType>
                     <xs:sequence>
                         <xs:element name="queue-full-behavior" type="wan-queue-full-behavior" minOccurs="0" maxOccurs="1"/>
+                        <xs:element name="initial-publisher-state" type="initial-publisher-state" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation>
+                                    Defines the initial state in which a WAN publisher is started.
```

Comment:
we can also state that state `ACTIVE` is the default one.

Reply:
Done.